pages/homepage.css

```diff
@@ -0,0 +1,155 @@
+:root {
+  --pink-color: #fc2678;
+  --bg-color: #402f27;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  width: 100%;
+}
+
+.navbar {
+  padding: 0;
+}
+
+.nav-link,
+.nav-link:hover {
+  color: var(--bg-color);
+}
+
+.active {
+  color: var(--bg-color);
+}
+
+.banner {
+  background-image: url("/assets/banner.png");
+  width: 100%;
+  height: 300px;
+  object-fit: cover;
+  margin-top: 1rem;
+}
+
+.image-btn {
+  margin-top: 13rem;
+  margin-left: 8rem;
```

Comment:
this can be adjusted with shorthand too.

Reply:
done